Contents.m

```diff
@@ -1,5 +1,5 @@
 % MATLAB-bindings for coupling library preCICE
-% Version 2.3.0.1 (R2018b, R2019a and R2019b) 18-Jan-2021
+% Version 2.4.0.0 (R2018b, R2019a and R2019b) 22-May-2022
```

Comment:
I guess the bindings are also working with newer versions. Can you add the MATLAB version that you are using for testing etc. here? We don't need to go for a complete list, but something more recent would be great.